app/models/proposal.rb

```diff
@@ -167,5 +167,11 @@ def restart
     Dispatcher.deliver_new_proposal_emails(self)
   end
 
+  def changed_fields comment_text
+    Comment.new({comment_text: comment_text, 
```

Comment:
I had to change the method for saving the comment to a manual approach to get it to have the right content in the email